pt/endpoints/powerdata/model.py

```diff
@@ -9,133 +11,113 @@
 
 class PowerData(db.Model, ETBaseMixin):
     __tablename__ = "powerdata"
-    uuid = db.Column(db.String(40), primary_key=True, unique=True, nullable=False)
     field = db.Column(db.String(120), unique=False, nullable=False)
     updated_at = db.Column(db.DateTime, unique=False, nullable=False)
     address = db.Column(db.String(120))
     # ForeignKey to History
-    history_id = db.Column(db.String(80), db.ForeignKey("history.uuid"), nullable=False)
+    history_id = db.Column(UUID(), db.ForeignKey("history.uuid"), nullable=False)
     history = db.relationship("History")
     data_type = db.Column(db.String(50))
     __mapper_args__ = {"polymorphic_identity": "Data", "polymorphic_on": data_type}
 
-    def __init__(self, data_struct):
-        self.uuid = data_struct["uuid"]
-        self.field = data_struct["field"]
-        self.updated_at = data_struct["updated_at"]
-        self.history_id = data_struct["history_id"]
-        self.address = data_struct["address"]
+    # def __init__(self, data_struct):
+    #     self.uuid = data_struct["uuid"]
+    #     self.field = data_struct["field"]
+    #     self.updated_at = data_struct["updated_at"]
+    #     self.history_id = data_struct["history_id"]
+    #     self.address = data_struct["address"]
 
```

Comment:
We can remove this section instead of commenting it.

Reply:
This is not the latest version, all comments have been remove :)